web/alt_text_manager.js

```diff
@@ -240,22 +242,20 @@ class AltTextManager {
     }
   }
 
-  #cancel() {
+  #close() {
     this.#eventBus.dispatch("reporttelemetry", {
       source: this,
       details: {
         type: "editing",
-        subtype: this.#currentEditor.editorType,
-        data: {
+        subtype: this.#currentEditor?.editorType,
```

Comment:
If the current editor is null or undefined, it doesn't make sense to send something.
However, in which case could it be null ? The dialog should have been opened with an editor.
Maybe when it's destroyed, but then it should be fixed thanks to https://github.com/mozilla/pdf.js/pull/17011.

Reply:
> However, in which case could it be null ? The dialog should have been opened with an editor.
> Maybe when it's destroyed, but then it should be fixed thanks to #17011.

Correct, after the mentioned PR it should always be defined now; I've removed the optional chaining.
Since I didn't know when and in which order the PRs would land I only wanted to prevent a (temporary) error here.